src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -52,6 +52,8 @@
 #include "ObjectMgr.h"
 #include "Opcodes.h"
 #include "OutdoorPvP.h"
+#include "MovementPacketSender.h"
+#include "MoveSplineInit.h"
```

Comment:
Why is this header neccessary here?

Reply:
it wasn't. it's the remnants of a rebase I assume. Anyway, I removed it.